android/app/src/main/res/drawable/system_splash.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+<item android:drawable="@drawable/ic_splash_system" android:gravity="center"/>
+</layer-list>
```

Comment:
Empty line

Reply:
There is no option to not use system splash screen
If we stop using ours we won't be able to display the info at the bottom - the system splash screen doesn't support this

I’d like to drop our splash screen, but I guess others aren’t ready for it yet 😄